test-network-nano-bash/configtx.yaml

```diff
@@ -343,45 +297,101 @@ Channel: &ChannelDefaults
 #
 ################################################################################
 Profiles:
-
-    TwoOrgsOrdererGenesis:
+    TwoOrgsChannel:
         <<: *ChannelDefaults
         Orderer:
             <<: *OrdererDefaults
             Organizations:
                 - *OrdererOrg
             Capabilities:
                 <<: *OrdererCapabilities
-        Consortiums:
-            SampleConsortium:
-                Organizations:
-                    - *Org1
-                    - *Org2
-    TwoOrgsChannel:
-        Consortium: SampleConsortium
-        <<: *ChannelDefaults
+            OrdererType: etcdraft
+            EtcdRaft:
+                Consenters:
+                - Host: 127.0.0.1
+                  Port: 6050
+                  ClientTLSCert: crypto-config/ordererOrganizations/example.com/orderers/orderer.example.com/tls/server.crt
+                  ServerTLSCert: crypto-config/ordererOrganizations/example.com/orderers/orderer.example.com/tls/server.crt
+                - Host: 127.0.0.1
+                  Port: 6051
+                  ClientTLSCert: crypto-config/ordererOrganizations/example.com/orderers/orderer2.example.com/tls/server.crt
+                  ServerTLSCert: crypto-config/ordererOrganizations/example.com/orderers/orderer2.example.com/tls/server.crt
+                - Host: 127.0.0.1
+                  Port: 6052
+                  ClientTLSCert: crypto-config/ordererOrganizations/example.com/orderers/orderer3.example.com/tls/server.crt
+                  ServerTLSCert: crypto-config/ordererOrganizations/example.com/orderers/orderer3.example.com/tls/server.crt
+                Options:
+                    TickInterval: 2500ms
+                    ElectionTick: 5
+                    HeartbeatTick: 1
+                    MaxInflightBlocks: 5
+                    SnapshotIntervalSize: 16 MB
         Application:
             <<: *ApplicationDefaults
             Organizations:
                 - *Org1
                 - *Org2
             Capabilities:
                 <<: *ApplicationCapabilities
-    Org1Channel:
-        Consortium: SampleConsortium
+    BFTTwoOrgsApplicationGenesis:
         <<: *ChannelDefaults
-        Application:
-            <<: *ApplicationDefaults
+        Orderer:
+            <<: *OrdererDefaults
+            Addresses:
+                - 127.0.0.1:6050
```

Comment:
We should not use the `Addresses` section, instead we should use the Endpoints:

https://github.com/hyperledger/fabric/blob/main/sampleconfig/configtx.yaml#L264-L270

https://github.com/hyperledger/fabric/blob/main/sampleconfig/configtx.yaml#L63

The addresses is global to all organization and is deprecated.

See: https://jira.hyperledger.org/browse/FAB-15662